### ISSUE TITLE
Telegram Interface

### ISSUE THREAD

**Issue by pavlovcik** [ubiquity/bounty-bot#289]
We should be able to access all of the bot commands from the Telegram interface, and not worry about spamming others. 

- We will need to associate GitHub IDs with Telegram IDs.

It will be useful to call the help menu there, and set payment multipliers.

---

It would be interesting experiment if users could the bot to do the entire flow of finding a bounty and assigning themselves via the bot reading the [DevPool directory](https://github.com/ubiquity/devpool/issues)

**Comment by Steveantor** [comment]
> ### Available commands
> ```
> - /assign: Assign the origin sender to the issue automatically.
> - /unassign: Unassign the origin sender from the issue automatically.
> - /help: List all available commands.
> - /payout: Disable automatic payment for the issue.
> - /wallet: <WALLET_ADDRESS | ENS_NAME>: Register the hunter's wallet address. 
>   ex1: /wallet 0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266 
>   ex2: /wallet vitalik.eth
> ```

`/payout` = request permit payout

**Comment by pavlovcik** [comment]
> Updated the wallet address for @Steveantor successfully! Your new address: `0x4841e8153a7b9E8B1F218E42d3cBaEb3e99C28eE`

That's funny that it parsed `assign` and Vitalik's wallet address. Perhaps its safer to set the help message to ubq.eth's wallet so that the funds wouldn't be lost. Alternatively we could set it to `0x0` in the help menu so that it is impossible to transfer to, guaranteeing no loss of funds.

**Comment by Steveantor** [comment]
> > Updated the wallet address for @Steveantor successfully! Your new address: `0x4841e8153a7b9E8B1F218E42d3cBaEb3e99C28eE`
> 
> That's funny that it parsed `assign` and Vitalik's wallet address from the help menu. Perhaps its safer to set the help message to ubq.eth's wallet so that the funds wouldn't be lost. Alternatively we could set it to `0x0` in the help menu so that it is impossible to transfer to, guaranteeing no loss of funds.

It's the issue with the parser, incorrectly parsing the comments

**Comment by pavlovcik** [comment]
I agree I think that the parser should be more strict. The entire comment for registration should be comprised only of the command and the wallet address.
